lib/saml/url_service.rb

```diff
@@ -91,6 +91,14 @@ def idme_url
       build_sso_url(build_authn_context(LOA::IDME_LOA1_VETS))
     end
 
+    def logingov_url
+      @type = 'logingov'
+      build_sso_url(build_authn_context(
+                      ['http://idmanagement.gov/ns/assurance/ial/1', 'http://idmanagement.gov/ns/assurance/aal/2'],
```

Comment:
Can we set these as constants, similar to the LOA module? `config/initializers/loa.rb`, but just an `IAL` module definition?